src/components/relationships-panel/RelationshipsPanel.vue

```diff
@@ -2,32 +2,45 @@
   <v-card flat id="relationships-panel">
     <div class="relationship-content" :style="{ backgroundColor: bgHex }">
       <v-row no-gutters class="align-center mt-5">
-
         <v-col cols="4">
-          <v-checkbox id="heir-legal-rep-checkbox" class="mt-0" v-model="selectedRelationships"
-            :value="RelationshipTypes.HEIR_LEGAL_REP" :label="RelationshipTypes.HEIR_LEGAL_REP"
-            :rules="relationshipRules" />
+          <v-checkbox id="heir-legal-rep-checkbox" class="mt-0"
+            v-model="selectedRelationships"
+            :value="RelationshipTypes.HEIR_LEGAL_REP"
+            :label="RelationshipTypes.HEIR_LEGAL_REP"
+            :error="displayErrorState"
+          />
```

Comment:
Trying a different approach here.  Instead of using the `rules` prop to show / hide validation errors, I'm using the `error` prop to show / hide the validation errors.  The difference is that using rules we add/remove the appropriate rules to show/hide the validation errors.  Using the `error` prop, we no longer change the rules dynamically, instead we show and hide the display of the validation errors.

Reply:
OK, thanks for trying something different :)

When does the component report that it's invalid?

I looked at the code below. There's a tiny bit of duplication but it looks so simple I'm happy with it.

Reply:
@severinbeauvais good question.  The component always reports its current validation status.  By using the `error` prop, I'm suggesting that we toggle on / off is whether the component displays the validation status to the user.